docs/migration-2.0.md

```diff
@@ -503,12 +503,6 @@ mongo.WithSession(context.TODO(),sess,func(ctx context.Context) error {
 
 ## Options Package
 
-The following fields were marked for internal use only and do not have replacement:
-
-- `ClientOptions.AuthenticateToAnything`
-- `FindOptions.OplogReplay`
-- `FindOneOptions.OplogReplay`
-
```

Comment:
It makes sense to remove OplogReplay from this list since it was added back to v2 with GODRIVER-3433, but AuthenticateToAnything was removed in v2 _because_ it was internalized. Users migrating that are relying on that field ([e.g.](https://sourcegraph.com/search?q=context:global+lang:Go++-repo:%5Egithub%5C.com/mongodb/mongo-go-driver%24+repo:has.content%28go.mongodb.org/mongo-driver%29+-file:%28%5E%7C/%29vendor/+AuthenticateToAnything&patternType=standard&case=yes&sm=0)) could be confused if it's removal is not called out in the migration guide. Suggest the following:

> `ClientOptions.AuthenticateToAnything` was marked for internal use in 1.x and does not have a replacement.